### PR TITLE
Get the reportback inbox closer to the mocks

### LINKED ISSUE #213: Make the inbox match the mocks
_FEATURE OVERVIEW_
-------------------
#### User Story
Let's actually do what @lkpttn designed! 3 columns and the right sizes and colors!

#### Additional Information (optional)
It should look like this:
screen_shot_2017-04-05_at_12 25 45_pm

### PR DESCRIPTION
**Pull request by katiecrane**
#### What's this PR do?
🎨 🎨 🎨 ❗️ 

This is what it looks like now:
image

1. There are two new button options to get the green and red buttons. 
2. There is an option to have a `container__block` be a third of the page instead of a half.
3. There is a fun new option called `quantity` that makes everything bold and purple and puts it in a purple box oOoOoOo

The other thing I did was to apply these new styles where they needed to be used.

#### How should this be reviewed?
🎨 👀 Did I CSS right?

#### Relevant tickets
Fixes #213

#### Checklist
- [ ] Tested on staging.

### CODE REVIEW

**Comment by sbsmith86** [comment]
AH! Haven't reviewed the code but it look gooooood from the screenshot!

File: resources/assets/app.scss [hunk 1]
```diff
@@ -24,3 +24,41 @@ $forge-path: "~@dosomething/forge/assets/";
...
+  }
+}
+
+.container__block.-third {
+  @include media($tablet) {
+    @include span(4 of 12);
+  }
+}
+
+.quantity {
```
Comment (DFurnes) [inline resources/assets/app.scss]: I think you could give this a `display: inline-block` to get more padding on the sides.

File: resources/assets/app.scss [hunk 2]
```diff
@@ -24,3 +24,41 @@ $forge-path: "~@dosomething/forge/assets/";
...
+  outline-color: #4e2b63;
```
Comment (DFurnes) [inline resources/assets/app.scss]: I think we want to use a `border` here instead of an outline.

File: resources/assets/app.scss (hunk 1 above)
Comment (katiecrane) [inline resources/assets/app.scss]: @DFurnes hmmm I added it but when I uncheck and check it in dev tools I don't see any difference 🤔

Comment (DFurnes) [inline resources/assets/app.scss]: css

**Comment by katiecrane** [comment]
@DFurnes updated! also the photo has been updated to reflect the changes

**Pull request merged**